instrumentation-core/spring/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'spring'
```

Comment:
instead of making this a root project, let's add this new module to the existing root project